src/mlpack/core/optimizers/cmaes/random.hpp

```diff
@@ -0,0 +1,90 @@
+#pragma once
+
+#include <ctime>
+#include <cmath>
+
+/**
+ * @class Random
+ * A pseudo random number generator.
+ */
+template<typename T>
+class Random
```

Comment:
Can you test this again with the ``Random`` function from ``core/math/random.hpp``?

It would be great if we could avoid the ``Random`` class; if someone likes to use another random seed for any mlpack method, they could just do : ``math::RandomSeed(seed);`` , which isn't possible with your code. Also the basic is something that is already implemented, so I don't see a good reason to keep it. What do you think?